shared/django_apps/legacy_migrations/migrations/legacy_sql/main/triggers/branches.py

```diff
@@ -1,32 +0,0 @@
-def run_sql(schema_editor):
-    schema_editor.execute(
-        """
-        create or replace function branches_update() returns trigger as $$
-        declare _ownerid int;
-        begin
-            -- update repos cache if main branch
-            update repos
-            set updatestamp = now(),
-                cache = update_json(cache::jsonb, 'commit', get_commit_minimum(new.repoid, new.head)::jsonb)
```

Comment:
two followup questions:
- is the `get_commit_minimum` function unused now? can we remove it?
- first time I see the `repos.cache`. what is its purpose?

Reply:
a) I'm not sure, but that should be contemplated in the commit trigger investigation
b) A field that got removed a while back, https://github.com/codecov/shared/blob/ac088a009c0c5cb7407752a1a9bcb7f3ccad1764/shared/django_apps/core/migrations/0034_remove_repository_cache.py#L20